translations/Polish/research/notes.md

```diff
@@ -0,0 +1,13 @@
+General notes:
+
+* The main difference between Polish and English writing are the special letters, in English, there is no ą, ć, ę, ł, ń, ś, ó, ż, ź.
```

Comment:
In general we try to keep line lengths at <= 76 characters, could you shorten them?